packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx

```diff
@@ -0,0 +1,21 @@
+import { BodyText, Button } from "../../design-system";
+
+type Props = {
+  isVerified: boolean;
+  onButtonClick: () => void;
+};
+
+export function MyOpenPool({ isVerified, onButtonClick }: Props) {
+  return (
+    <div className="grid place-items-center bg-green-100 pt-[60px] pb-[48px]">
+      <BodyText size="normal" className="text-dark-50">
+        {isVerified
+          ? "You haven’t created your pool yet, create your first pool"
+          : "To create the first pool, you need to verify Identity"}
```

Comment:
Sorry, I know design wrote this. Can you change it to:

"Please create your first pool."

"Please verify your identity to create a pool"